Dockerfile.amd64-alpine

```diff
@@ -0,0 +1,42 @@
+FROM resin/amd64-alpine-node:0.10.44-slim
+
+# Supervisor apt dependencies
+RUN apk add --update \
+		btrfs-progs \
+		ca-certificates \
+		rsync \
+		supervisor \
+	&& rm -rf /var/cache/apk/*
+
+# Copy supervisord configuration files
+COPY config/supervisor/ /etc/supervisor/
+
+# Install dependencies
+WORKDIR /app
+COPY package.json postinstall.sh /app/
+RUN set -x \
+	&& buildDeps='g++ sqlite-dev sqlite-libs make' \
+	&& apk add --update $buildDeps \
+	&& JOBS=MAX npm install --unsafe-perm --production --no-optional \
+	&& cd /app/node_modules/sqlite3 && ./node_modules/.bin/node-pre-gyp install --build-from-source --sqlite=/usr/lib \
```

Comment:
@pcarranzav, I've tried and had no issue at all. PR updated!
Jenkins builds passed!
https://jenkins.dev.resin.io/job/amd64-alpine-supervisor-gh/13/
https://jenkins.dev.resin.io/job/armhf-alpine-supervisor-gh/25/
https://jenkins.dev.resin.io/job/i386-alpine-supervisor-gh/7/